compiler/rustc_borrowck/src/region_infer/mod.rs

```diff
@@ -791,6 +791,9 @@ impl<'tcx> RegionInferenceContext<'tcx> {
         member_constraint_index: NllMemberConstraintIndex,
         choice_regions: &[ty::RegionVid],
     ) -> bool {
```

Comment:
two thoughts:

- the `bool` is unused by the only caller, can change the return type to `()`.
- can you add a comment that this only adds the member constraints to the region graph and that we don't check them here but instead in `check_member_constraints` after the region graph has been computed?